ir_attachment_conversion_queue_job/models/ir_attachment.py

```diff
@@ -34,3 +34,30 @@ def convert_image_attachments(self, attachment):
                 attachment.res_id,
                 repr(e),
             )
+
+    def _product_image_job_queue(self):
+        mapping = {
+            "product.image": "image",
+            "product.public.category": "image",
+        }
```

Comment:
I think we might as well include the images of the partners/users.

How about images on the website?